### PR TITLE
v1.0.0->v1.0.1

### PR DESCRIPTION
**Pull request by yangj1211**
## What type of PR is this?

- [ ] Enhancement
- [ ] Displaying
- [ ] Typo
- [x] Doc Request

## Which issue(s) this PR fixes:

issue #

### CODE REVIEW

**Comment by yangj1211** [comment]
@dengn need to review

File: docs/MatrixOne/Get-Started/install-on-linux/install-on-linux-method1.md [hunk 1]
```diff
@@ -184,7 +184,7 @@ GCC_VERSION="8.5.0"
 CLANG_VERSION="13.0"
 GO_VERSION="1.20"
 MO_GIT_URL="https://github.com/matrixorigin/matrixone.git"
-MO_DEFAULT_VERSION="1.0.0-rc1"
+MO_DEFAULT_VERSION="v1.0.0"
```
Comment (dengn) [inline docs/MatrixOne/Get-Started/install-on-linux/install-on-linux-method1.md:187]: 这里也应该是1.0.1

File: docs/MatrixOne/Get-Started/install-on-macos/install-on-macos-method1.md [hunk 2]
```diff
@@ -168,7 +168,7 @@ GCC_VERSION="8.5.0"
 CLANG_VERSION="13.0"
 GO_VERSION="1.20"
 MO_GIT_URL="https://github.com/matrixorigin/matrixone.git"
-MO_DEFAULT_VERSION="1.0.0-rc1"
+MO_DEFAULT_VERSION="v1.0.0"
```
Comment (dengn) [inline docs/MatrixOne/Get-Started/install-on-macos/install-on-macos-method1.md:171]: 这里也应该是1.0.1

File: package-lock.json [hunk 3]
```diff
@@ -0,0 +1,2855 @@
+{
```
Comment (dengn) [inline package-lock.json:1]: remove this unrelated file

**Review by dengn — CHANGES_REQUESTED** [review]

**Comment by yangj1211** [comment]
@dengn need to review

**Review by dengn — APPROVED** [review]

**Pull request merged**